src/Altinn.Platform/Altinn.Platform.Authorization/Authorization/Controllers/DelegationsController.cs

```diff
@@ -82,6 +85,64 @@ public async Task<ActionResult> Post([FromBody] List<Rule> rules)
             }
         }
 
+        /// <summary>
+        /// Endpoint for retrieving delegated rules between parties
+        /// </summary>
+        /// <response code="400">Bad Request</response>
+        /// <response code="500">Internal Server Error</response>
+        [HttpPost]
+        ////[Authorize(Policy = AuthzConstants.ALTINNII_AUTHORIZATION)]
```

Comment:
Autorisasjonen på fjernes utkommentering på

Reply:
Uncommented that line